sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -255,29 +263,26 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
    *   before the session lock expires, then the message lands back in the Queue/Subscription for the next
    *   receive operation.
    *
+   * @param options - Options bag to pass an abort signal or tracing options.
    * @returns Promise<Date> - New lock token expiry date and time in UTC format.
    * @throws Error if the underlying connection or receiver is closed.
    * @throws MessagingError if the service returns an error while renewing session lock.
    */
-  async renewSessionLock(): Promise<Date> {
+  async renewSessionLock(options?: OperationOptions): Promise<Date> {
     this._throwIfReceiverOrConnectionClosed();
-    const retryOptions = this._sessionReceiverOptions.retryOptions || {};
-    retryOptions.timeoutInMs = getRetryAttemptTimeoutInMs(retryOptions);
 
     const renewSessionLockOperationPromise = async () => {
       await this._createMessageSessionIfDoesntExist();
       this._messageSession!.sessionLockedUntilUtc = await this._context.managementClient!.renewSessionLock(
-        this.sessionId!,
-        undefined,
-        retryOptions.timeoutInMs!
+        this.sessionId!
```

Comment:
Shouldn't this have options passed through as well?
(I saw your note about renewLock coming in a different PR because it's for message settlement but I think renewSessionLock is on the session itself)

Reply:
Yup, you right. Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8159/commits/bec36f46590c86e808c415292694c6270e10bd52